### DISCUSSION TITLE
Add Logo Token

### DISCUSSION THREAD

**Discussion by avenbreaks** [blockscout/blockscout#9122] [Q&A] [answered]
# Add Logo Token

How do I add a logo to every token that has been minted or just minted in our blockchain explorer? 

Explorer: https://tokioscan-v2.engram.tech

**Comment by GimluCom** [comment]
Are you asking how to change the default icon for all tokens, or assign icons to individual tokens?

**Reply by avenbreaks** [reply]
assign icons to individual tokens, like USDT logo, USDC logo, ETH logo, etc on my token list explorer.

**Reply by GimluCom** [reply, marked as answer]
Assuming your chain does not have the tokens listed on Coingecko or the Trustwallet repository:
You could put them in a Github repository with the same structure as https://github.com/trustwallet/assets
Then you can modify this code to point to your repository https://github.com/blockscout/blockscout/blob/v6.0.0-beta/apps/block_scout_web/assets/js/lib/token_icon.js#L1-L22

**Reply by ghost** [reply]
It would be very useful if we would be able to pass logo URL via env var
- Run own chain
- Deploy a token
- Upload the logo
- Pass logo via variable and restart BE

**Comment by avenbreaks** [comment]
@Mojmir-R @vbaranov
